custom_components/e3dc_rscp/coordinator.py

```diff
@@ -49,8 +51,13 @@ def __init__(self, hass: HomeAssistant, config_entry: ConfigEntry) -> None:
             hass, _LOGGER, name=DOMAIN, update_interval=timedelta(seconds=10)
         )
 
+    def get_e3dcconfig(self) -> dict:
```

Comment:
Please move this further down, I'd like the connect call - it is HAs primary entry point - at the top of the file for clarity.

Reply:
Done